sdk/storage/azfile/file/examples_test.go

```diff
@@ -0,0 +1,281 @@
+	shareClient, err := share.NewClientFromConnectionString(connectionString, shareName, nil)
+	handleError(err)
+
+	_, err = shareClient.Create(context.Background(), nil)
+	handleError(err)
+
+	fileClient := shareClient.NewRootDirectoryClient().NewFileClient(fileName)
+	_, err = fileClient.Create(context.Background(), 5, nil)
+	handleError(err)
+
+	data := []byte{'h', 'e', 'l', 'l', 'o'}
+	err = fileClient.UploadBuffer(context.Background(), data, nil)
+	handleError(err)
+
+	_, err = fileClient.Delete(context.Background(), nil)
+	handleError(err)
+
+	_, err = shareClient.Delete(context.Background(), nil)
+	handleError(err)
+}
+
+func Example_fileClient_UploadStream() {
+	// Your connection string can be obtained from the Azure Portal.
+	connectionString, ok := os.LookupEnv("AZURE_STORAGE_CONNECTION_STRING")
+	if !ok {
+		log.Fatal("the environment variable 'AZURE_STORAGE_CONNECTION_STRING' could not be found")
+	}
+	shareName := "testShare"
+	fileName := "testFile"
+	shareClient, err := share.NewClientFromConnectionString(connectionString, shareName, nil)
+	handleError(err)
+
+	_, err = shareClient.Create(context.Background(), nil)
+	handleError(err)
+
+	fileClient := shareClient.NewRootDirectoryClient().NewFileClient(fileName)
+	_, err = fileClient.Create(context.Background(), 5, nil)
+	handleError(err)
+
+	err = fileClient.UploadStream(
+		context.TODO(),
+		streaming.NopCloser(strings.NewReader("Some text")),
+		nil,
+	)
+	handleError(err)
+
+	_, err = fileClient.Delete(context.Background(), nil)
+	handleError(err)
+
+	_, err = shareClient.Delete(context.Background(), nil)
+	handleError(err)
+}
+
+func Example_fileClient_UploadAndClearRange() {
+	// Your connection string can be obtained from the Azure Portal.
+	connectionString, ok := os.LookupEnv("AZURE_STORAGE_CONNECTION_STRING")
+	if !ok {
+		log.Fatal("the environment variable 'AZURE_STORAGE_CONNECTION_STRING' could not be found")
+	}
+	shareName := "testShare"
+	fileName := "testFile"
+	shareClient, err := share.NewClientFromConnectionString(connectionString, shareName, nil)
+	handleError(err)
+
+	_, err = shareClient.Create(context.Background(), nil)
+	handleError(err)
+
+	fileClient := shareClient.NewRootDirectoryClient().NewFileClient(fileName)
+	_, err = fileClient.Create(context.Background(), 5, nil)
+	handleError(err)
+
+	contentR, _ := generateData(5)
+
+	_, err = fileClient.UploadRange(context.Background(), 0, contentR, nil)
+	handleError(err)
+
+	rangeList, err := fileClient.GetRangeList(context.Background(), nil)
+	handleError(err)
+	fmt.Println(rangeList.Ranges)
+
+	_, err = fileClient.ClearRange(context.Background(), file.HTTPRange{Offset: 0, Count: int64(5)}, nil)
+	handleError(err)
+
+	rangeList2, err := fileClient.GetRangeList(context.Background(), nil)
+	handleError(err)
+
+	fmt.Println(rangeList2.Ranges, 0)
+	_, err = fileClient.Delete(context.Background(), nil)
+	handleError(err)
+
+	_, err = shareClient.Delete(context.Background(), nil)
+	handleError(err)
+}
```

Comment:
I see that the examples are missing for these methods. Can you please add those.
DownloadStream, DownloadBuffer, DownloadFile
UploadFile
GetSASURL
UploadRangeFromURL
Resize, SetHTTPHeaders
StartCopyFromURL, AbortCopy